### PR TITLE
Add jim configuration

### PR DESCRIPTION
**Pull request by cekees**
Preloading the lustre library prevents a warning about not being able to load it. It doesn't appear to change how things run on that system.

# Mandatory Checklist

Please ensure that the following criteria are met:

- [x] Title of pull request describes the changes/features
- [x] Request at least 2 reviewers
- [x] If new files are being added, the files are no larger than 100kB. Post the file sizes.
- [x] Code coverage did not decrease. If this is a bug fix, a test should cover that bug fix. If a new feature is added, a test should be made to cover that feature.
- [x] New features have appropriate documentation strings (readable by sphinx)
- [x] Contributor has read and agreed with CONTRIBUTING.md and has added themselves to CONTRIBUTORS.md 

As a general rule of thumb, try to follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines.

# Description

### CODE REVIEW

**Comment by cekees** [comment]
@zhang-alvin this new stack built fine on the buildbot, but it ran out of memory compiling with of the adaption submodules earlier. Could you take a quick lock at the zoltan and scorec versions and make sure I haven't gone backwards or something? This build issues seems vaguely familiar.

**Comment by zhang-alvin** [comment]
@cekees, yes the Travis error:
`/usr/bin/ld: out of memory allocating 281474976710656 bytes after a total of 3100672 bytes`
was "fixed" by not caching the zoltan package built by the buildbots. I recall from forums that this happens if there package is built on a 32-bit platform but run on a 64-bit platform.

File: proteus/config/jim.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+from .default import *
+    
+#platform_extra_compile_args = ['-DPETSC_INCLUDE_AS_C']
+    
+PROTEUS_PRELOAD_LIBS=['/p/home/apps/hpe/mpt-2.21/lib/libmpi.so','/opt/hpe/hpc/mpt/mpt-2.21/lib/libmpi_lustre.so']
+#PROTEUS_PRELOAD_LIBS=['/p/home/apps/sgi/mpt-2.12-sgi712r26/lib/libmpi.so',
```
Comment (zhang-alvin) [inline proteus/config/jim.py:7]: Are the commented-out lines going to be useful?

Comment (cekees) [reply proteus/config/jim.py:7]: Only as a record to help somebody in the future trying to find the kinds of places where they've put these libraries. It's in the history, though, so I'll remove it.

**Comment by cekees** [comment]
OK, thanks @zhang-alvin, I removed the zoltan artifact from the cache.

**Comment by cekees** [comment]
Closing; Merged into #1190 to fix last test failures there.

**Pull request closed without merging**